### PR TITLE
Tagger conglomerator (for fedora-hubs)

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

File: fedmsg_meta_fedora_infrastructure/tests/conglomerate/tagger/test_tags.py [hunk 1]
```diff
@@ -0,0 +1,1134 @@
+import fedmsg.tests.test_meta
+
+
+class TestTaggerConglomerateByUser(
+        fedmsg.tests.test_meta.ConglomerateBase):
+    expected = []
+    expected =[
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/tests/conglomerate/tagger/test_tags.py]: Declared twice?

File: fedmsg_meta_fedora_infrastructure/conglomerators/tagger/tags.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import fedmsg.meta.base
+from fedmsg_meta_fedora_infrastructure.fasshim import avatar_url
+
+class AbstractTaggerConglomerator(fedmsg.meta.base.BaseConglomerator):
+
+    def merge(self, constituents, **config):
+        ms = constituents  # shorthand
+
+        user = constituents[0]['msg']['user']['username']
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/conglomerators/tagger/tags.py]: Here we use ``constituents``, below we use ``ms``, we're not being very consistent here

**Comment by pypingou** [comment]
Couple of comments but no blockers (although I would like to harmonize the use of ``ms``/``constituents``).

File: fedmsg_meta_fedora_infrastructure/tests/conglomerate/tagger/test_tags.py (hunk 1 above)
Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/tests/conglomerate/tagger/test_tags.py]: Oop!  Good catch.

**Pull request merged**